src/test/java/de/muenchen/isi/domain/service/AbfrageServiceTest.java

```diff
@@ -38,7 +39,8 @@ class AbfrageServiceTest {
     private final AbfrageDomainMapper abfrageDomainMapper = new AbfrageDomainMapperImpl(
             new AbfragevarianteDomainMapperImpl(
```

Comment:
würde ich in einer Zeile schreiben. Das ist aber Geschmackssache.

Reply:
Der Grund für die Aufgliederung in mehrere Zeilen ist, dass der Mapper `AbfrageDomainMapperImpl` zwei Mapper im Ctor benötigt, welcher wiederum einen Mapper im Ctor benötigt.